packages/documentation-spa/docs/microservices/search/search-service.md

```diff
@@ -89,11 +89,11 @@ In the GraphQL GET Operations are defined as Queries and POST/PUT/PATCH operatio
 
 ### Search Query
 
-- Operation Name - search
-    - Supported Query Variables - query, start, rows
+- Operation Name - Search
+- Supported Query Variables - ```query```, ```start```, ```rows```
 
 Example Query
-```graphql
+```sh
```

Comment:
This adds proper syntax highlighting in github markdown for graphql queries.
```suggestion
```graphql
```

Reply:
Changed 👍🏻